src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="section">
+
+        <legend>Personal information</legend>
+
+        <label class="form__field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form__field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form__field">
+            How old are You?
```

Comment:
should be two spaces, fit in all places
```suggestion
        <label class="form__field">
          How old are You?
```